README.md

```diff
@@ -1,22 +1,79 @@
 # Ape Chainstack Plugin
 
-Chainstack Provider plugins for networks
+Chainstack Provider plugins for networks.
 
-## Dependencies
+This plugin allows using the ApeWorX framework with Chainstack as a node provider in an easy and integrated way.
 
-* [python3](https://www.python.org/downloads) version 3.7.2 or greater, python3-dev
+ApeWorX is an innovative smart contract development and testing framework. It is inspired by Brownie, and it has essentially the same syntax. Still, ApeworX focuses on a more modular approach, allowing us to build and use external plugins to add functionality.
```

Comment:
nit, but our non-existent style guide is to use newlines per sentence in markdown

```suggestion
ApeWorX is an innovative smart contract development and testing framework.
It is inspired by Brownie, and it has essentially the same syntax.
Still, ApeworX focuses on a more modular approach, allowing us to build and use external plugins to add functionality.
```